plugin/postgres/plugin.go

```diff
@@ -141,6 +143,13 @@ func (p PostgresPlugin) Validate(endpoint ShieldEndpoint) error {
 		ansi.Printf("@G{\u2713 pg_password}  @C{%s}\n", s)
 	}
 
+	s, err = endpoint.StringValue("pg_database")
+	if err != nil {
+		ansi.Printf("@R{\u2717 pg_database  %s}\n", err)
```

Comment:
I *think* this should be using StringValueDefault, similar to port, so that if this is not specified, it doesn't cause an error, but does cause an if the value is not a string. @jhunt will know for sure though